velox/dwio/dwrf/test/E2EFilterTest.cpp

```diff
@@ -276,7 +276,7 @@ TEST_F(E2EFilterTest, timestamp) {
       "long_val:bigint",
       [&]() {},
       false,
-      {"long_val"},
+      {"timestamp_val"},
```

Comment:
Can you keep `long_val` but add `timestamp_val`?  Also can you change the `wrapInStruct` parameter to true so that we test the cases for nested fields as well?